frontend/src/scenes/experiments/experimentsLogic.ts

```diff
@@ -1,21 +1,23 @@
-import { kea } from 'kea'
+import { actions, connect, events, kea, path, reducers, selectors } from 'kea'
 import api from 'lib/api'
 import type { experimentsLogicType } from './experimentsLogicType'
 import { teamLogic } from 'scenes/teamLogic'
-import { Experiment, ExperimentsTabs, ExperimentStatus } from '~/types'
+import { AvailableFeature, Experiment, ExperimentsTabs, ExperimentStatus } from '~/types'
 import { lemonToast } from 'lib/lemon-ui/lemonToast'
 import Fuse from 'fuse.js'
 import { userLogic } from 'scenes/userLogic'
+import { subscriptions } from 'kea-subscriptions'
+import { loaders } from 'kea-loaders'
 
-export const experimentsLogic = kea<experimentsLogicType>({
-    path: ['scenes', 'experiments', 'experimentsLogic'],
-    connect: { values: [teamLogic, ['currentTeamId'], userLogic, ['user']] },
-    actions: {
+export const experimentsLogic = kea<experimentsLogicType>([
+    path(['scenes', 'experiments', 'experimentsLogic']),
+    connect({ values: [teamLogic, ['currentTeamId'], userLogic, ['user']] }),
```

Comment:
```suggestion
    connect({ values: [teamLogic, ['currentTeamId'], userLogic, ['user', 'hasAvailableFeature']] }),
```

Reply:
not sure how that was working previously without that there then 😆